src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/integration/TestDataIntTest.kt

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.integration
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.TestData
+
+class TestDataIntTest : IntegrationTestBase() {
+  @Test
+  fun `Get a list of test data items`() {
+    val result = webTestClient.get()
+      .uri("/test/data")
+      .accept(MediaType.APPLICATION_JSON)
+      .headers(setAuthorisation(roles = listOf("ROLE_CVL_ADMIN")))
+      .exchange()
```

Comment:
Will need your own role here - not sure what this will be yet so one for discussion.

Reply:
I've changed this to a placeholder, but certainly one to discuss!